### ISSUE TITLE
Question about the infer result

### ISSUE THREAD

**Issue by vixlott** [1YCxZ/ECM-seq2seq#5]
Hi. After training for 15,000 steps, I found out the infer results are too short. No matter what the source are, the results are always "哈哈"，“你”，"我", or “嗯”. 
The loss is decreasing but I am looking for better and longer results. Could you please tell me why this problem happens? 
Thanks.

**Comment by 1YCxZ** [comment]
What is the train perplexity at 15,000 steps? In general, when the perplexity drops to around 30, you can get a good reply. And, If the tgt in the training set is very short, the quality of the response obtained by infer will not be very good.

**Comment by vixlott** [comment]
The perplexity is around 65. I used a portion of the STC-3 as my dataset. There are totally 22,000 pairs.
I think the quality of the dataset is good. Somehow, after 25,000 steps, the model does not converge (loss is nan). At the 25,000 steps, the loss is around 55.

**Comment by 1YCxZ** [comment]
Try LSTM and set learning rate to 0.0001
